tests/stub/shared.py

```diff
@@ -328,8 +328,8 @@ def get_requests(self, pattern, silence_period=0.1):
             # lines start with something like "10:08:33  [#EBE0>#2332]  "
             # plus some color escape sequences and ends on a newline
             line = re.sub(r"\x1b\[[\d;]+m", "", line[:-1])
-            line = re.sub(r"^\d{2}:\d{2}:\d{2}\s+\[[0-9A-Fa-f#>]+\]\s+", "",
-                          line)
+            line = re.sub(r"^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3}\s+"
```

Comment:
Should this be separated by a full stop from seconds like `YYYY-MM-dd HH:mm:ss.SSS` like in ISO8601??

Reply:
I first thought the logging API didn't provide enough extensibility to do that. I had a second look and made it work. I also decided to omit the date because we're not running stub tests over days or even just hours. Thanks for raising this.